tests/api_fastapi/core_api/routes/ui/test_assets.py

```diff
@@ -47,5 +47,5 @@ def test_next_run_assets(test_client, dag_maker):
     assert response.status_code == 200
     assert response.json() == {
         "asset_expression": {"all": ["s3://bucket/key/1"]},
-        "events": [{"id": 17, "uri": "s3://bucket/key/1", "lastUpdate": None}],
+        "events": [{"id": 20, "uri": "s3://bucket/key/1", "lastUpdate": None}],
```

Comment:
Why does this change?

Reply:
By the time the fixture `test_client` is used, the example DAGs are loaded.

https://github.com/apache/airflow/blob/28d0a7e90a5a56bc20ba582a0af355f53504aa28/tests/api_fastapi/core_api/routes/ui/test_assets.py#L38

and it loads the example DAG here

https://github.com/apache/airflow/blob/28d0a7e90a5a56bc20ba582a0af355f53504aa28/airflow/www/extensions/init_dagbag.py#L29

because `include_examples` is not set and then falls back to `conf.getboolean("core", "LOAD_EXAMPLES")`